docs/guide/application-side/session-access.md

```diff
@@ -102,7 +102,7 @@ Time at which the session was last refreshed, either `undefined` if no refresh w
 
 Returns the current Cross Site Request Forgery Token (CSRF Token) required to make POST requests (e.g. for signing in and signing out).
 
-You likely only need to use this if you are not using the built-in `signIn()` and `signOut()` methods. Read more: https://next-auth.js.org/getting-started/client#getcsrftoken
+You likely only need to use this if you are not using the built-in `signIn()` and `signOut()` methods. Read more: <https://next-auth.js.org/getting-started/client#getcsrftoken>
```

Comment:
Why did you change the link format? This is a markdown file, and links are correctly recognized by Vitepress
```suggestion
You likely only need to use this if you are not using the built-in `signIn()` and `signOut()` methods. Read more: https://next-auth.js.org/getting-started/client#getcsrftoken
```

Reply:
Ahh, that is probably markdownlint. I highly recommend to implement this. But I will remove them for now.

Reply:
It is this rule: https://github.com/DavidAnson/markdownlint/blob/v0.36.1/doc/md034.md

Reply:
https://github.com/sidebase/nuxt-auth/pull/702/commits/7739fbc1d06753c7f69649aab4011347fa6a43a2